test/python/qobj/test_pulse_converter.py

```diff
@@ -207,13 +207,12 @@ def setUp(self):
     def test_drive_instruction(self):
         """Test converted qobj from PulseInstruction."""
         cmd = self.linear
-        instruction = cmd(DriveChannel(0)) << 10
+        instruction = cmd(DriveChannel(0))
```

Comment:
```suggestion
        instruction = self.linear(DriveChannel(0))
```
why not?
There also seems to be other locations in this file where SamplePulse instances are named `command` see e.g. `test_drive_instruction`. Given the changes `pulse` might be a better name.
Can you go over this file and check if there are other things that need to be updated to reflect the current methodology?